### PR TITLE
Changed to AdcInterrupts behavior

### LINKED ISSUE #248: ADC Interrupts Vector is wrong on STM32F103
This block from `modm/src/modm/platform/adc/stm32/adc_interrupts.cpp.in` is wrong on STM32F103.
```cpp
extern "C" void
ADC_IRQHandler(void)
{
```

The real name of this handler is
```cpp
extern "C" void
ADC1_2_IRQHandler(void)
```

### PR DESCRIPTION
**Pull request by asmfreak**
Fixes #248 
This fix is quite dumb. It does not add any logic to enabling/disabling intrerupts

### CODE REVIEW

File: src/modm/platform/adc/stm32/adc_shared_interrupts.cpp.in [hunk 1]
```diff
@@ -14,18 +14,22 @@
...
+%% for id in shared_irq_ids
+%% if id in instances
 #include "adc_interrupt_{{ id }}.hpp"
+%% endif
 %% endfor
 
+%% for name, irqs in shared_irqs.items()
 extern "C" void
-ADC_IRQHandler(void)
+{{ name }}_IRQHandler(void)
```
Comment (salkinium) [inline src/modm/platform/adc/stm32/adc_shared_interrupts.cpp.in]: Also use `MODM_ISR` here

**Review by salkinium — DISMISSED** [review]
Otherwise a good fix, the enable/disable were never part of the API, so it's fine to obmit them

File: src/modm/platform/adc/stm32/adc_shared_interrupts.cpp.in (hunk 1 above)
Comment (asmfreak) [reply src/modm/platform/adc/stm32/adc_shared_interrupts.cpp.in]: Done

**Comment by asmfreak** [comment]
> Otherwise a good fix, the enable/disable were never part of the API, so it's fine to obmit them

I meant that I didn't change any logic in Adc class in `enableInterruptVector(const uint32_t priority, const bool enable = true)`

**Review by salkinium — APPROVED** [review]
Thanks!

**Comment by salkinium** [comment]
Btw, GitHub supports pull request drafts natively now: https://github.blog/2019-02-14-introducing-draft-pull-requests/

**Comment by asmfreak** [comment]
@salkinium That's a really nice feature, I didn't know about it!

**Pull request merged**